WinImageBuilder.psm1

```diff
@@ -1461,6 +1488,7 @@ function New-WindowsFromGoldenImage {
         Copy-Item $ConfigFilePath "$resourcesDir\config.ini"
         Set-WindowsWallpaper -WinDrive $driveLetterGold -WallpaperPath $windowsImageConfig.wallpaper_path `
             -WallpaperSolidColor $windowsImageConfig.wallpaper_solid_color
+        Download-SDelete $resourcesDir ([string]$image.ImageArchitecture)

```

Comment:
ditto

Reply:
Huh? Thats not true! Just one line below is the Download-Cloudinit call which uses exactly the same.

Reply:
Oops yes, thats in this Golden thingy, you are right

Reply:
Just to clarify:
In New-WindowsCloudImage, $image DOES exist
In New-WindowsFromGoldenImage, $image DOES NOT exist. Instead, $imageInfo.imageArchitecture must be used.

Reply:
got it, you are correct ;)